modules/nf-core/vclust/cluster/meta.yml

```diff
@@ -0,0 +1,103 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "vclust_cluster"
+description: Vclust cluster performs threshold-based clustering by assigning a
+  genome sequence to a cluster if its similarity (e.g., ANI) to the cluster
+  meets or exceeds a user-defined threshold.
+keywords:
+  - cluster
+  - virus
+  - filter
+  - contig
+  - scaffold
+  - phage
+tools:
+  - "vclust":
+      description: "Fast and accurate tool for calculating ANI and clustering virus
+        genomes and metagenomes."
```

Comment:
Why is this split across two lines? If yes then I would use the `|` thingy to push to the multiple-line paragraph  thing